Glossary.md

```diff
@@ -0,0 +1,149 @@
+
+---
+
+
+## Civilian Accounts
+**Civilian Accounts**:  User accounts that are not associated with any special roles or permissions beyond standard user functionality. By contrast, System Accounts and Node Accounts are specialized accounts that have specific roles and responsibilities within the infrastructure and governance of the Hedera network.
+
+## Config.txt
+**Config.txt**: A text file loaded by the platform on startup if no valid state is found on disk. This file contains the address book to use when starting from genesis.
+
+## Congestion Pricing
+**Congestion Pricing**: A mechanism designed to manage network congestion by dynamically adjusting transaction fees based on network demand. The primary goal of congestion pricing is to discourage excessive network usage during peak times.
+
+## Consensus Time
+**Consensus Time**: The timestamp assigned to a transaction once it has reached consensus across the network. This timestamp indicates the moment when the network agrees on the order and validity of the transaction, ensuring that all participants have a consistent view of the transaction sequence.
+
+---
+
+## Delegate Call
+**Delegate Call**: Used to call a function in another contract, but with the context of the calling contract. This allows for the execution of functions in one contract as if they were part of another but retains the original caller’s context (msg.sender and msg.value).
+
+---
+
+## EOA
+**EOA (Externally Owned Accounts)**: Accounts that are controlled by private keys and are not associated with smart contracts. These accounts are used to send and receive transactions on the network and interact with smart contracts. EOAs are typically owned by individual users and are used to manage their assets and participate in the network.
+
+---
+
+## Hollow Accounts
+**Hollow Accounts**: An account that has been created with an account number and alias but lacks an account key. This term is specifically used to describe accounts generated through the Auto Account Creation feature, which allows applications to create user accounts instantly, even without an internet connection, by assigning an account alias.
+
+---
+
+## Leaky Tests
+**Leaky Tests**: Are unit or other tests that should not be run in parallel because they can “leak” some of their state or data out into the global state.
+
+---
+
+## Precompiles
+**Precompiles**: Contracts that are compiled and deployed on the blockchain before they are executed. They are used to optimize the execution of certain operations that are computationally expensive or require a large amount of gas to execute. Precompiled contracts are typically used to perform cryptographic operations, such as hashing, encryption, and decryption, that are required for the operation of the blockchain.
```

Comment:
This is not quite it.  In Ethereum chains they are things that _behave as if they were contracts_ deployed at specific addresses (in Ethereum, they are very low addresses, `0x00` .. `0x0F` or so).  They aren't _implemented_ as contracts because that's far too slow - they're built-in to the code of the Ethereum EVM, which checks to see if the address you're calling is one of the precompiles and then just runs the related code.
* With one exception: The EIP-4788 "beacon roots contract" which _is_ a real contract deployed at a specific address ...

We call our equivalent "system contracts".  But that's a relatively recent decision we made and thus you'll find, in our codebase, a bunch of references to "precompiles" - including in names of packages/classes/methods - that are still there.  Plus in older documentation.

Anyway, maybe refer to "System Contracts" you defined below in order to provide the contrast.